### PR TITLE
Benchmarks: Remove unused dimensions, use somewhat more iterations.

### PR DESCRIPTION
**Pull request by gianm**

### CODE REVIEW

**Comment by gianm** [comment]
```
sbt.ResolveException: download failed: org.apache.flink#flink-shaded-hadoop2;1.0.0!flink-shaded-hadoop2.jar
```

**Comment by fjy** [comment]
👍

File: core/src/test/scala/com/metamx/tranquility/benchmark/BenchmarkMain.scala [hunk 1]
```diff
@@ -50,6 +49,7 @@ import com.twitter.util.Future
 import com.twitter.util.NonFatal
 import com.twitter.util.Return
 import com.twitter.util.Throw
+import io.druid.data.input.InputRow
```
Comment (drcrallen) [inline core/src/test/scala/com/metamx/tranquility/benchmark/BenchmarkMain.scala]: unused?

**Comment by drcrallen** [comment]
Minor comment but 👍

File: core/src/test/scala/com/metamx/tranquility/benchmark/BenchmarkMain.scala (hunk 1 above)
Comment (gianm) [inline core/src/test/scala/com/metamx/tranquility/benchmark/BenchmarkMain.scala]: removed

**Pull request merged**